ethereum/p2p/build.gradle

```diff
@@ -42,6 +42,8 @@ dependencies {
   implementation 'dnsjava:dnsjava'
   implementation 'io.prometheus:simpleclient'
   implementation 'io.vertx:vertx-core'
+  implementation 'io.netty:netty-transport-native-unix-common'
```

Comment:
I think the order of dependencies is mostly alphabetical. If that's the case this should go above `prometheus`

Reply:
tnx reordered both, I thought spotless would have taken care of them